contrib/rpm/crio.spec

```diff
@@ -21,6 +21,7 @@ Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcomm
 Provides:       %{repo}
 
 BuildRequires:  golang-github-cpuguy83-go-md2man
+Requires: container-storage-setup >= 0.3
```

Comment:
Now we have new version of container-storage-setup (0.6) and that does not conflict with older versions. It might be a good idea to bump up the minimum version requirement to 0.6 instead.